server/sonar-server/src/main/java/org/sonar/server/measure/ws/ComponentTreeDataLoader.java

```diff
@@ -128,6 +130,16 @@ ComponentTreeData load(ComponentTreeWsRequest wsRequest) {
     }
   }
 
+  private ComponentDto loadComponent(DbSession dbSession, ComponentTreeWsRequest request) {
+    String componentKey = request.getComponent();
+    String componentId = request.getBaseComponentId();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "getBaseComponentId"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)